### PR TITLE
Update serving guide including api_server and gradio

### PR DESCRIPTION
**Pull request by lvhan028**

### CODE REVIEW

File: docs/en/serving/gradio.md [hunk 1]
```diff
@@ -1,18 +1,33 @@
...
+
+All it takes is one-line command, with the `{model_path}` replaced by the model ID from huggingface hub, such as `internlm/internlm2-chat-7b`, or the local path to the model.
+
+For detailed parameters of the command, please turn to `lmdeploy serve gradio -h` for help.
+
+## Create a huggingface demo
+
+If you want to create an online demo project for your model on huggingface, please follow the steps below.
+
+## Step 1: Create space
```
Comment (RunningLeon) [inline docs/en/serving/gradio.md:18]: may put it as the third level: `###`

File: docs/en/serving/gradio.md [hunk 2]
```diff
@@ -1,18 +1,33 @@
...
 
 First, register for a Hugging Face account. After successful registration, click on your profile picture in the upper right corner and select “New Space” to create one. Follow the Hugging Face guide to choose the necessary configurations, and you will have a blank demo space ready.
 
-## A demo for LMDeploy
+## Step 2: Develop demo's entrypoint `app.py`
```
Comment (RunningLeon) [inline docs/en/serving/gradio.md:22]: may put it as the third level: `###`

File: docs/zh_cn/serving/restful_api.md [hunk 3]
```diff
@@ -57,28 +103,35 @@ for item in api_client.chat_completions_v1(model=model_name, messages=messages):
...
 
-LMDeploy 的 `/v1/chat/interactive` api 支持将对话内容管理在服务端，但是我们默认关闭。如果想尝试，请阅读以下介绍：
+关于 `/v1/chat/interactive` 接口，我们默认是关闭的。在使用时，请设置`interactive_mode = True`打开它。否则，它会退化为 openai 接口。
 
-- 交互模式下，对话历史保存在 server。在一次完整的多轮对话中，所有请求设置`interactive_mode = True`, `session_id`保持相同 (不为 -1，这是缺省值)。
-- 非交互模式下，server 不保存历史记录。
-
-交互模式可以通过 `interactive_mode` 布尔量参数控制。下面是一个普通模式的例子，
-如果要体验交互模式，将 `interactive_mode=True` 传入即可。
+在交互式推理中，每个对话序列的 id 必须唯一，所有属于该徐立的对话请求，必须使用相同的 id。这里的 id 对应与接口中的 `session_id`。
```
Comment (AllentDan) [inline docs/zh_cn/serving/restful_api.md:114]: 徐立？

Comment (lvhan028) [reply docs/zh_cn/serving/restful_api.md]: “独立”，已修改

**Review by AllentDan — APPROVED** [review]

**Review by RunningLeon — APPROVED** [review]
LGTM

**Pull request merged**